fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/bitmap/ComposedBitmapIndexAggregate.java

```diff
@@ -82,9 +83,9 @@
      * @return an {@code Optional} query plan or {@code Optional.empty} if planning is not possible
      */
     @Nonnull
-    public static Optional<ComposedBitmapIndexQueryPlan> tryPlan(@Nonnull RecordQueryPlanner planner,
-                                                                 @Nonnull RecordQuery query,
-                                                                 @Nonnull IndexAggregateFunction indexAggregateFunction) {
+    public static Optional<RecordQueryPlan> tryPlan(@Nonnull RecordQueryPlanner planner,
```

Comment:
I believe that an earlier review of this class suggested that this was a more appropriate signature. It didn't matter then, but it does now.

Reply:
I guess this class doesn't have an API stability annotation. It kind of feels like it should have been marked experimental, but alas. Not sure how much of a stickler we want to be here. My inclination here is to just let the signature be changed.

Reply:
Somehow half the classes in this package are annotated and the other half is not. I made them all experimental, which I believe was the intention.

Reply:
Yeah, that sounds right